temporallib/client/client.py

```diff
@@ -70,6 +70,7 @@ async def reconnect_loop(self):
         """
         Reconnects to the Temporal server periodically when the token expires.
         """
+        backoff = self._initial_backoff
```

Comment:
If the first call to `self._reconnect()` raises an exception, `backoff` is not defined.